RealmTasks Apple/RealmTasks Shared/SharedFunctionality.swift

```diff
@@ -49,7 +48,7 @@ private func setDefaultRealmConfiguration(with user: SyncUser) {
         let itemsReference = ThreadSafeReference(to: items)
         // Deduplicate
         DispatchQueue(label: "io.realm.RealmTasks.bg").async {
-            let realm = try! Realm()
+            let realm = try! Realm(configuration: realm.configuration)
```

Comment:
this change isn't necessary, but is makes this function more semantically correct and future-proof in case we ever change the default Realm.